x-pack/test/functional/services/machine_learning/navigation.ts

```diff
@@ -50,6 +50,10 @@ export function MachineLearningNavigationProvider({
       ]);
     },
 
+    async navigateToOverview() {
+      await this.navigateToArea('mlTabOverview', 'mlPageOverview');
```

Comment:
If you follow my suggestion from above, also change the tab data subject to `mlMainTab overview` here.